features/facebookImport/src/model/analyses/utils/on-off-facebook-data-restructuring.js

```diff
@@ -116,3 +116,24 @@ export function buildDisplayData(
 
     return getIntoChartStructure(displayData);
 }
+
+export function top5OffFacebookCompanies(facebookAccount) {
+    return [...facebookAccount.offFacebookCompanies]
+        .map((e) => {
+            return { title: e.name, count: e.events.length };
+        })
+        .sort((a, b) => b.count - a.count);
+}
+
+export function offFacebookActivityTypes(facebookAccount) {
```

Comment:
There already should be `groupOffFacebookEventsByType` that we were using before. Could that be used/improve instead?

Reply:
Done